content/construction/index.md

```diff
@@ -9,79 +9,40 @@ weight = 30
+North Bayshore became Google headquarters starting in 2003, and over time the company added about 50 office buildings. The City Council realized that the area needed to be redeveloped to support a much larger population, so by 2014, the City's planners developed the North Bayshore Precise Plan. This plan defined their vision for the new neighborhood, including areas to be developed for residential housing, and those that needed to be maintained as parks and environmental habitats.
 
-### Construction in Progress
+### A New City
 
-​
-But more important is the status of the construction. We will be working with Pear project manager Jonah Elisha going forward, and Superintendent Matt Li Marzi, but we still talk to Tim Steele occasionally about the project.
+Over the last decade, we have seen major changes. Many new residential projects have been planned and approved by the City, and in the next decade, we are going to see a small city​ growing up around us. 
 
-Tim told us that they have placed vibration monitors in the park to record any potential disturbances to our homes. If your home has been leveled and you have the paperwork to prove it, they will restore any leveling changes or damage done by construction. But because they are using auger technology instead of jack-hammers for excavating, they think any such disturbances will be minimized.
-​
-Devcon Construction is the builder, and the South Parcel development started with the new Google office building on the corner of Inigo and La Avenida first. There will eventually be a light at that corner, and CalTrans will be reconfiguring the 101 offramp to divert traffic into that area, which should ease traffic on Shoreline.
-​
-Sobrato has a construction office in the Pear Theatre office building, but we can phone Jonah Elisha at (808) 747-3462 to talk about any concerns about their project.
+There are four major developments being planned by The Sobrato Organization, Eden Housing, Google Real Estate and Workplace Services, and Sywest Development LLC.
 
-### Google and Sobrato Construction
+#### Sobrato's Mixed-Use Residential Development
 
-We have been involved with Google planners and developers of North Bayshore since 2015. Google Real Estate and Workplace Services, Government Affairs and Public Policy, Lendlease, and Community Development experts have been helpful and responsive to Santiago Villa residents.
-​
-With Sobrato and Google's new developments, we should have a vibrant new neighborhood by 2025. We're going to continue to work with them to make sure our community remains an integrated part of the new neighborhood.
+The Sobrato Organization has been keeping SVNA informed of their plans for two large developments in front of our park since 2017. They have sought and received our feedback on the development of those projects, one of which is now near completion. 
```

Comment:
```suggestion
The Sobrato Organization has been keeping SVNA informed of their plans for two large developments in front of our park since 2017. They have sought and received our feedback on the development of those projects, one of which is now nearing completion (Summer 2024). 
```